public/app/features/expressions/ExpressionQueryEditor.tsx

```diff
@@ -44,7 +44,7 @@ function useExpressionsCache() {
       // We want to use the same value for Reduce, Resample and Threshold
       case ExpressionQueryType.reduce:
       case ExpressionQueryType.resample:
-      case ExpressionQueryType.resample:
+      case ExpressionQueryType.threshold:
```

Comment:
Did this catch a bug? 🤔

Reply:
I think so - but I wasn't able to get into a breakpoint in this code so I can't say for sure 🤷 Regardless, I think it should go in